layouts/_default/official-cve-feed.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "https://jsonfeed.org/version/1.1",
+  "title": "Auto-refreshing Official CVE Feed",
+  "home_page_url": "https://kubernetes.io",
+  "feed_url": "https://kubernetes.io/docs/reference/issues-security/official-cve-feed/index.json",
+  "description": "Auto-refreshing official CVE feed for Kubernetes repository",
+  "author": "Kubernetes Community",
+  "items": [
+  {{ range $i, $e := getJSON "https://storage.googleapis.com/fake-test-cve-feed-kep-3203/official-cve-feed.json" }}
```

Comment:
This doesn't look right for the final piece. Should we mark this PR as a work-in-progress?

Reply:
Would `getJSON "URL_GOES_HERE" | jsonify` also work?

Reply:
Yes this bucket creation is pending: https://github.com/kubernetes/k8s.io/pull/4009

On the other comment, I am curious what would be the benefit of adding `| jsonify` to the line? 

If I add it it is failing with this error:
`Rebuild failed: failed to render pages: render of "page" failed: "/src/layouts/_default/cve-feed.json:9:115": execute of template failed at <jsonify>: range can’t iterate over `